pkg/services/featuremgmt/registry.go

```diff
@@ -662,5 +662,12 @@ var (
 			Owner:           grafanaOperatorExperienceSquad,
 			RequiresRestart: true,
 		},
+		{
+			Name:         "awsAsyncQueryCaching",
```

Comment:
Have you completed an [FRR](https://docs.google.com/document/d/1QL9Ly8KnXzpb6ISbg49pTODRO5mhA5tkkfIZVX6pqQU/edit#heading=h.nmhrirqphdnu) for this and shared with the saas team? Just want to ensure that they are aware of new experimental features going into Cloud 😄

Reply:
Good catch, I'll write one and make sure they see it before I merge.